WhateverGreen/kern_igfx.hpp

```diff
@@ -110,6 +111,7 @@ class IGFX {
 	 *  Framebuffer hard-code patch
 	 */
 	FramebufferCFL framebufferPatch {};
+	uint32_t fPatchCursorMemorySize;	// Haswell only... and therefore not present in FramebufferCFL
```

Comment:
Please restyle the comment like the one above. Do not want to be picky, but it is important to keep the codebase as clean as possible for own good.